client/state/login/magic-login/actions.js

```diff
@@ -70,9 +81,58 @@ export const fetchMagicLoginRequestEmail = email => dispatch => {
 	} );
 };
 
+export const authError = error => {
+	return {
+		type: MAGIC_LOGIN_REQUEST_AUTH_ERROR,
+		error,
+	};
+};
+
+export const authSuccess = () => {
+	return {
+		type: MAGIC_LOGIN_REQUEST_AUTH_SUCCESS,
+	};
+};
+
+export const fetchMagicLoginAuthenticate = ( email, token, tt ) => dispatch => {
+	dispatch( { type: MAGIC_LOGIN_REQUEST_AUTH_FETCH } );
+
+	const postData = {
+		client_id: config( 'wpcom_signup_id' ),
+		client_secret: config( 'wpcom_signup_key' ),
+		email,
+		token,
+		tt,
+	};
+
+	request
```

Comment:
Are you envisioning adding this to wpcom-undocumented, or does it not make sense for this application?